src/config.py

```diff
@@ -21,4 +22,7 @@
 RUN_PDF_PARSER = os.getenv("RUN_PDF_PARSER", True)
 RUN_HTML_PARSER = os.getenv("RUN_HTML_PARSER", True)
 
+# Default set by trial and error based on behaviour of the parsing model
+PDF_N_PROCESSES = int(os.getenv("PDF_N_PROCESSES", multiprocessing.cpu_count() / 2))
```

Comment:
I remember this briefly from the meeting but out of interest why is the number of processes cpu count / 2?

Reply:
not sure - was from some analysis that @joel-wright and @stefl14 did